jkq/ddsim/hybridqasmsimulator.py

```diff
@@ -0,0 +1,170 @@
+"""Backend for DDSIM Hybrid Schrodinger-Feynman Simulator."""
+
+import logging
+import time
+import uuid
+from math import log2
+import warnings
+from typing import Union, List
+
+from qiskit.providers import BackendV1, Options
+from qiskit import QiskitError, QuantumCircuit
+from qiskit.providers.models import BackendConfiguration, BackendStatus
+from qiskit.qobj import QasmQobjExperiment, Qobj, QasmQobj, PulseQobj
+from qiskit.result import Result
+from qiskit.compiler import assemble
+from qiskit.utils.multiprocessing import local_hardware_info
+
+from .jkqjob import JKQJob
+from .jkqerror import JKQSimulatorError
+from jkq import ddsim
+
+logger = logging.getLogger(__name__)
+
+
+class HybridQasmSimulator(BackendV1):
+    """Python interface to JKQ DDSIM Hybrid Schrodinger-Feynman Simulator"""
+
+    SHOW_STATE_VECTOR = False
+
+    @classmethod
+    def _default_options(cls) -> Options:
+        return Options(
+            shots=None,
+            parameter_binds=None,
+            simulator_seed=None,
+            mode="amplitude",
+            nthreads=local_hardware_info()['memory']
```

Comment:
Number of theads is the amount of memory? Is that correct?

Reply:
Ups. Good catch.